ambry-store/src/test/java/com.github.ambry.store/BlobStoreTest.java

```diff
@@ -280,6 +283,66 @@ public void cleanup() throws InterruptedException, IOException, StoreException {
   }
 
   /**
+   * Tests blob store use of {@link ClusterManagerWriteStatusDelegate}
+   * @throws StoreException
+   */
+  @Test
+  public void testClusterManagerWriteStatusDelegateUse() throws StoreException, IOException, InterruptedException {
+    //Test only relevant for stores with segmented logs
+    assumeTrue(isLogSegmented);
```

Comment:
Isn't the RW -> RO is relevant to the non-segmented store? The second part of the test can be skipped if non-segmented (RO -> RW)

Reply:
I thought RW -> RO happened already before compaction was even a factor... what was the workflow for that before?

Reply:
nm explained in person that currently RW -> RO conversions are manual